src/test/java/domain/CarsTest.java

```diff
@@ -0,0 +1,27 @@
+package domain;
+
+import domain.car.Car;
+import domain.car.Cars;
+import domain.name.Name;
+import domain.name.NameCatalog;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+public class CarsTest {
+    @Test
+    @DisplayName("자동차 이름 목록을 통해 자동차 목록을 포함한 일급객체를 만든다.")
```

Comment:
```suggestion
    @DisplayName("자동차 이름 목록을 통해 자동차 목록을 포함한 일급컬랙션를 만든다.")
```
일급 객체(first-class object)은 다른 의미로 사용되어요.

Reply:
꼼꼼히 리뷰 해주셔서 감사합니다🙇‍♂️